src/query/join.rs

```diff
@@ -107,6 +107,15 @@ where
         }
         select_two
     }
+
+    /// Inner Join with a Related Entity and select both Entity.
```

Comment:
My only nit pick might be to move this slightly upwards, just below `find_also_related`